src/commands/moderation/Ban.ts

```diff
@@ -60,14 +56,13 @@ export default class Ban {
     }
 
     try {
+      await modEmbed(
+        { interaction, user, action: "Banned", duration, dm: true, dbAction: "BAN", expiresAt },
```

Comment:
Ok but if they have their dm disabled, would the ban just fail every time ? Is not being able to send a dm considered as an error ?

Reply:
if they don't have dms it'll just return, banning then dming will simply never work since the person isn't in the server